Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -159,4 +157,13 @@ public class FieldNpc : Actor<Npc> {
         SendControl = false;
         Remove(delay: (int) (Value.Metadata.Dead.Time * 1000));
     }
+
+    public virtual void Animate(string sequenceName, float duration = -1f) {
+        if (!Value.Animations.TryGetValue(sequenceName, out AnimationSequence? sequence)) {
+            Logger.Error("Invalid sequence: {Sequence} for npc {NpcId}", sequenceName, Value.Metadata.Id);
+            CurrentRoutine = new WaitRoutine(this, IdleSequence.Id, 1f);
+        }
```

Comment:
there should be a return here I think?

Reply:
that's what hours of debugging do, brain go smush